### ISSUE TITLE
provide the possibility to override getSchema() in AbstractJacksonKafkaAvroDeserializer

### ISSUE THREAD

**Issue by valentin-braun** [productboardlabs/jackson-kafka-avro-serializer#144]
First of all, thank you @lukas-krecan for this brilliant project. You did a great job.

In my special use case of getting rid of the schema registry, I would be able to override the `getSchema(int schemaId)` in `AbstractJacksonKafkaAvroDeserializer` (see [L67-L69](https://github.com/productboardlabs/jackson-kafka-avro-serializer/blob/master/src/main/java/io/github/productboardlabs/kafka/serializers/AbstractJacksonKafkaAvroDeserializer.java#L67-L69)). But for now this method is private and delegates directly to `schemaRegistry.getSchemaById()`.

```java
private Schema getSchema(int schemaId) throws IOException, RestClientException {
    return ((io.confluent.kafka.schemaregistry.avro.AvroSchema) schemaRegistry.getSchemaById(schemaId)).rawSchema();
}
```

It would be very helpful to delegate to the public method `getSchemaById(schemaId)` of the super class `AbstractKafkaSchemaSerDe` instead, which can be overridden if necessary.
```java
private Schema getSchema(int schemaId) throws IOException, RestClientException {
    return ((io.confluent.kafka.schemaregistry.avro.AvroSchema) super.getSchemaById(schemaId)).rawSchema();
}
```

Is there a chance to get this improvement into in the next release version?

**Comment by lukas-krecan** [comment]
Hi, can you please send a PR?

**Comment by valentin-braun** [comment]
Sure, there it is: https://github.com/productboardlabs/jackson-kafka-avro-serializer/pull/145

**Comment by lukas-krecan** [comment]
Released as 0.9.0 Thanks

**Comment by valentin-braun** [comment]
@lukas-krecan I made a mistake. The method `getSchemaById(schemaId)` should be called with `this` instead of `super` so that it can be overridden in the concrete subclass if necessary.

Here is the fix: [#148](https://github.com/productboardlabs/jackson-kafka-avro-serializer/pull/148)

Can you build a release 0.9.2?

**Comment by lukas-krecan** [comment]
Done, thanks
